frontend/packages/shared/src/api/mutations.ts

```diff
@@ -1,29 +1,5 @@
 import { del, get, post, put } from 'app-shared/utils/networking';
-import {
-  appMetadataAttachmentPath,
-  copyAppPath,
-  createRepoPath,
-  deploymentsPath,
-  formLayoutNamePath,
-  formLayoutPath,
-  layoutSetsPath,
-  layoutSetPath,
-  layoutSettingsPath,
-  releasesPath,
-  repoCommitPath,
-  repoCommitPushPath,
-  repoPushPath,
-  repoResetPath,
-  ruleConfigPath,
-  textResourceIdsPath,
-  textResourcesPath,
-  userLogoutPath,
-  userStarredRepoPath,
-  datamodelPath,
-  resourcePolicyPath,
-  resourceCreatePath,
-  resourceEditPath, datamodelAddXsdFromRepoPath, createDatamodelPath
-} from 'app-shared/api/paths';
+import { appMetadataAttachmentPath, copyAppPath, createRepoPath, deploymentsPath, formLayoutNamePath, formLayoutPath, layoutSetsPath, layoutSetPath, layoutSettingsPath, releasesPath, repoCommitPath, repoCommitPushPath, repoPushPath, repoResetPath, ruleConfigPath, textResourceIdsPath, textResourcesPath, userLogoutPath, userStarredRepoPath, datamodelPath, resourcePolicyPath, resourceCreatePath, resourceEditPath, datamodelAddXsdFromRepoPath, createDatamodelPath, appPolicyPath } from 'app-shared/api/paths';
```

Comment:
No idea why it formatted like this, but I guess it has to do with my prettier settings. I find it weird that it behaves like this, because on all other files it formats it the other way :/

Reply:
I found the issue and fixed it within this PR, https://github.com/Altinn/altinn-studio/pull/10878